lib/platform/vsts/index.js

```diff
@@ -57,7 +57,10 @@ async function initRepo(repoName, token, endpoint) {
   config.fileList = null;
   config.prList = null;
   const repos = await gitApi().getRepositories();
-  const repo = repos.filter(c => c.name === repoName)[0];
+  const names = vstsHelper.getProjectAndRepo(repoName);
+  const repo = repos.filter(
+    c => c.name === names.repo && c.project.name === names.project
```

Comment:
This should probably be a case insensitive string compare.

Reply:
I just checked in VSTS and yes we cannot create `repo1` and `Repo1`. So it make sense to check in lowerCase.

I'll update this tonight